food-truck-api/src/main/java/food/truck/api/endpoint/TruckEndpoint.java

```diff
@@ -251,8 +280,8 @@ public void deleteTruckRouteLocations(@AuthenticationPrincipal User user, @Reque
     }
 
     @GetMapping("/truck/{truckId}/active-route")
-    public Route getTodaysRoute(@PathVariable long truckId) {
-        return truckService.getActiveRoute(truckId, LocalDateTime.now().getDayOfWeek());
+    public Route getTodaysRoute(@PathVariable long truckId, @RequestParam LocalDateTime now) {
```

Comment:
If we store times internally as UTC, like above, then we can get "now" from the backend anywhere: 
var now = OffsetDateTime.now(ZoneOffset.UTC);
Then we don't have to pass in "now" all over the place

Reply:
See my comment on the first issue.